Tests/New-ProGetFeed.Tests.ps1

```diff
@@ -0,0 +1,115 @@
+
+& (Join-Path -Path $PSScriptRoot -ChildPath 'Initialize-Tests.ps1' -Resolve)
+
+function Initialize-ProGetFeedTests
+{
+    [CmdletBinding()]
+    param(
+        [Parameter(Mandatory=$true)]
+        [object]
+        $ProGetSession
+    )
+    
+    $Global:Error.Clear()
+    
+    # Remove all feeds from target ProGet instance
+    Invoke-PGNativeApiMethod -Session $ProGetSession -Name 'Feeds_GetFeeds' -Parameter @{IncludeInactive_Indicator = $true} |
+        ForEach-Object { 
+            Invoke-PGNativeApiMethod -Session $ProGetSession -Name 'Feeds_DeleteFeed' -Parameter @{Feed_Id = $PSItem.Feed_Id}
+        }
+
+
+}
+
+Describe 'New-ProGetFeed.create a new Universal package feed' {
+    
+    $session = New-ProGetTestSession
+    Initialize-ProGetFeedTests -ProGetSession $session
+
+    New-ProGetFeed -ProGetSession $session -FeedType 'ProGet' -FeedName 'Apps'
+    $feedExists = Get-ProGetFeed -ProGetSession $session -FeedType 'ProGet' -FeedName 'Apps'
+    
+    It 'should write no errors' {
+        $Global:Error | Should BeNullOrEmpty
+    }
+    
+    It 'should create a new ProGet universal feed' {
+        $feedExists | Should Be $true
+    }
+}
+
+Describe 'New-ProGetFeed.when attempting to create a duplicate package feed' {

```

Comment:
Tried to implement this suggestion, but PowerShell throws an error, writes it to the $Global:Error variable, and reverts execution to -ErrorAction Continue. Searching online, this seems like a known bug when using `ErrorAction Ignore` with advanced functions..

NotSupportedException: The value Ignore is not supported for an ActionPreference va
riable. The provided value should be used only as a value for a preference parameter, 
and has been replaced by the default value. For more information, see the Help topic, 
"about_Preference_Variables."
   at New-ProGetFeed, C:\Build\ProGetAutomation\ProGetAutomation\Functions\New-ProGetF
eed.ps1: line 52
   at <ScriptBlock>, C:\Build\ProGetAutomation\Tests\New-ProGetFeed.Tests.ps1: line 66Leave a comment